### PR TITLE
[Enhancement] Reduce log printing

### PR DESCRIPTION
**Pull request by sduzh**
Fixes #issue

## What type of PR is this:
- [ ] BugFix
- [ ] Feature
- [x] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Checklist:
- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.1
  - [ ] 3.0
  - [ ] 2.5
  - [ ] 2.4

### CODE REVIEW

**Review by luohaha — APPROVED** [review]

**Comment by luohaha** [comment]
Maybe do this after this feature became stable?

**Review by wyb — DISMISSED** [review]

File: be/src/storage/lake/update_manager.cpp [hunk 1]
```diff
@@ -167,7 +167,7 @@ Status UpdateManager::publish_primary_key_tablet(const TxnLogPB_OpWrite& op_writ
     builder->apply_opwrite(op_write);
     cost_str << " [apply meta] " << watch.elapsed_time();
 
-    LOG(INFO) << strings::Substitute(
+    VLOG(2) << strings::Substitute(
```
Comment (luohaha) [inline be/src/storage/lake/update_manager.cpp:170]: This log can remain INFO?

**Comment by sduzh** [comment]
> Maybe do this after this feature became stable?

The primary key will be released soon in version 3.1, so I thought it was already a stable feature.

File: be/src/storage/lake/update_manager.cpp (hunk 1 above)
Comment (sduzh) [reply be/src/storage/lake/update_manager.cpp]: Reverted

**Review by kevincai — APPROVED** [review]

**Review by gengjun-git — APPROVED** [review]

**Review by wyb — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-3.1

**Pull request merged**